server/app/controllers/fusor/api/customer_portal/customer_portal_proxies_controller.rb

```diff
@@ -41,6 +53,50 @@ def is_authenticated
+        param :login, String, required: true, desc: 'Subscription account login username'
+        def get_owners; end #apipie docs dummy.  Routes to get()
+
+        api :get, '/pools', 'Get a list of subscription pools from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        def get_pools; end #apipie docs dummy.  Routes to get()
+
+        api :get, '/owners/:id/consumers', 'Get a list of subscription consumers from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        param :id, :identifier, required: true, desc: 'ID of the subscription owner'
+        def get_consumers; end #apipie docs dummy.  Routes to get()
+
+        api :get, '/consumers/:id', 'Get a subscription consumer from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        param :id, :identifier, required: true, desc: 'Subscription consumer UUID'
+        def get_consumer; end #apipie docs dummy.  Routes to get()
+
+        api :post, '/consumers', 'Create a new subscription consumer from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        param :name, String, desc: 'Name of the new subscription management application'
+        param :type, String, desc: 'Type of the new subscription management application (ex. satellite)'
+        param :facts, Hash, desc: 'Facts about the new subscription management application' do
+          param :distributor_version, String, desc: 'Distributor version of the new subscription management application (ex. sat-6.2)'
+          param 'system.certificate_version', String, desc: 'System certificate version of the new subscription management application (ex. 3.2)'
+        end
+        def create_consumer; end #apipie docs dummy.  Routes to post()
+
+        api :get, '/consumers/:id/entitlements',
+            'Get a list of subscription entitlements from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        param :id, :identifier, desc: 'Subscription consumer UUID'
+        def get_entitlements; end #apipie docs dummy.  Routes to get()
+
+        api :post, '/consumers/:id/entitlements',
+            'Create new subscription entitlement from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        param :id, :identifier, required: true, desc: 'Subscription consumer UUID'
+        def create_entitlement; end #apipie docs dummy.  Routes to post()
+
+        api :delete, '/consumers/:id/entitlements',
+            'Delete all entitlements for a subscription consumer from RHN customer portal. See customer portal API documentation for a full list of parameters.'
+        param :id, :identifier, required: true, desc: 'Subscription consumer UUID'
+        def delete_entitlements; end #apipie docs dummy.  Routes to delete()
+        #### END Apipie Docs ####
+
```

Comment:
WOW! That's heinous. Sucks when you go against the grain with ruby tools huh? :ok: